Tensile/Configs/test_sgemm.yaml

```diff
@@ -82,6 +82,33 @@ BenchmarkProblems:
         - ProblemSizes:
           - Range: [ [8, 32, 0, 75], [8, 32, 0, 75], [3], [63, 1, 64] ]
 
+    - # BenchmarkProblemSizeGroup - Source
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - LoopTail: [True]
+        - EdgeType: ["ShiftPtr"]
+      ForkParameters:
+        - KernelLanguage: ["Source", "Assembly"]
```

Comment:
This is both source and assembly - intentional?

Reply:
This one is run nightly. I left both "Source" and "Assembly" in the tests that were moved to nightly.